sdv/demo.py

```diff
@@ -325,6 +325,11 @@ def _load_tabular_dummy():
     faker = Faker()
     names = [faker.name() for _ in range(12)]
     adresses = [faker.address() for _ in range(12)]
+    start_date = datetime(1980, 1, 1)
+    start_dates = [
+        start_date + timedelta(days=np.random.randint(0, 14600))
```

Comment:
I think this field choice is delicate, because we will then need to make the `years_in_the_company` match the `start_date`. Maybe we could use a date which is not related to for how long the employee has been in the company?

Reply:
good point